src/paths.ts

```diff
@@ -4,3 +4,4 @@ export const HEALTHCHECK_URI = `${ROOT_URI}/healthcheck`
 
 // Pages
 export const FORM_PAGE_URI = `${ROOT_URI}/form`
+export const WHOTOTELL_URI = `${ROOT_URI}/who-to-tell`
```

Comment:
Perhaps WHO_TO_TELL_URI would be a better name?

Reply:
Ok, This is what Carl named it so I just copied what he had - I'll let him know to change it too! :)